src/lighteval/pipeline.py

```diff
@@ -85,16 +91,22 @@ class PipelineParameters:
     use_chat_template: bool = False
     system_prompt: str | None = None
 
-    def __post_init__(self):
+    def __post_init__(self):  # noqa C901
         if self.launcher_type == ParallelismManager.ACCELERATE:
             if not is_accelerate_available():
                 raise ImportError(NO_ACCELERATE_ERROR_MSG)
```

Comment:
Do you remember how we differentiate between accelerate and nothing needed for // (for example for local runs)?

Reply:
We never did, before this refacto we always used `accelerate` for backend management (except when using nanotron)

Reply:
Yes but we did not always require accelerate to be installed even when launching with ligtheval accelerate ^^

Reply:
(I know I did a bunch of launchs in a local env without accelerate for ex)

Reply:
Mhh just launch lighteval from a fresh install on main. It does not work, lighteval accelerate will require `accelerate` to be install all the time.

What command did you use ?

Reply:
It was lighteval accelerate blablabla, but maybe the option was removed at some point